sites/dwhdelft.nl/components/JoinOptions.vue

```diff
@@ -17,11 +20,14 @@ en:
 nl:
   title: Kennismaken met <strong>DWH</strong>?
   description: |-
-    Je bent altijd welkom op een zaterdagavond tijdens een baravond. Vind je dit spannend? Dan kun je
-    gebruik maken van ons barbuddysysteem!
-  outsite_hint: |-
-    Ben je 28 jaar of jonger? Kijk dan ook eens bij
-    <a href="https://outsite.nl/" class="font-bold">Outsite</a>, het jongerendeel van de vereniging.
+    De beste manier om DWH te leren kennen is bij één van onze activiteiten. Kom langs, of meld je 
+    aan voor een barbuddy zodat je gelijk al iemand kent. Ben je 28 of jonger? Kijk dan ook eens bij 
+    <a href="https://outsite.nl/" class="font-bold">Outsite (28-)</a> of 
+    <a href="https://jongenoutdelft.nl/" class="font-bold">Jong&Out Delft (12-18)</a>.
```

Comment:
```suggestion
    <a href="/jongenout" class="font-bold">Jong&Out Delft (12-18)</a>.
```
dan is de navigatie mooier